### PR TITLE
Fix post release tests

### PR DESCRIPTION
**Pull request by tianfeng92**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -357,41 +357,38 @@ jobs:
...
+  post-release-macos-tests:
+    runs-on: ubuntu-latest
+    needs: publish-release
+    env:
+      SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+      SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Deps
+        run: npm ci
```
Comment (alexplischke) [inline .github/workflows/release.yml:380]: Why do we need to install dependencies at this point? 🤔

Comment (alexplischke) [reply .github/workflows/release.yml:380]: Oh, because of `saucectl` itself.

Comment (alexplischke) [reply .github/workflows/release.yml:380]: Imho, it'd be better if you just install `saucectl` on the fly and nothing else at this point. Faster certainly.

Comment (tianfeng92) [reply .github/workflows/release.yml:380]: sounds better.

**Review by alexplischke — APPROVED** [review]

**Pull request merged**